pkg/virt-operator/resource/generate/components/prometheus.go

```diff
@@ -491,6 +491,20 @@ func NewPrometheusRuleSpec(ns string, workloadUpdatesEnabled bool) *v1.Prometheu
 				operatorHealthImpactLabelKey: "none",
 			},
 		},
+		{
+			Alert: "WinVirtualMachineCRCErrors",
+			Expr:  intstr.FromString("kubevirt_vm_pvc_info * on (name) group_left() kubevirt_vm_info{os=~'windows.*'} * on(persistentvolumeclaim) kube_persistentvolumeclaim_info{storageclass='ocs-storagecluster-ceph-rbd'}"),
```

Comment:
Indeed, I don't think this is a reliable check. A cluster can have multiple StorageClasses that have some specific configuration, independent of the optimization/improvement options for Windows VMs. Admins that have created additional StorageClasses for a different set of storage configurations (NVMe devices only, erasure-coded replication, ...) will not trigger the alert, even when they should.

The best way is to check if the PVC for the VM has `mapOptions` with `rxbounce` in the parameters of the PV.

A sort of check like this would do. I don't know how to put this in a Prometheus rule.
```
if pv.spec.csi.volumeAttributes.mounter == "rbd" && pv.spec.csi.volumeAttributes.mapOptions !~= "krbd:rxbounce" {
   alert()
}
```